plugins/node/plugin.zsh

```diff
@@ -1,18 +1,32 @@
 # Color definitions
-GEOMETRY_COLOR_NODE_NPM_VERSION=${GEOMETRY_COLOR_NODE_NPM_VERSION:-black}
+GEOMETRY_COLOR_PACKAGER_VERSION=${GEOMETRY_COLOR_PACKAGER_VERSION:-black}
 
 # Symbol definitions
-GEOMETRY_SYMBOL_NODE_NPM_VERSION=${GEOMETRY_SYMBOL_NODE_NPM_VERSION:-"⬡"}
-GEOMETRY_NODE_NPM_VERSION=$(prompt_geometry_colorize $GEOMETRY_COLOR_NODE_NPM_VERSION $GEOMETRY_SYMBOL_NODE_NPM_VERSION) 
+GEOMETRY_SYMBOL_PACKAGER_VERSION=${GEOMETRY_SYMBOL_PACKAGER_VERSION:-"⬡"}
+GEOMETRY_NODE_PACKAGER_VERSION=$(prompt_geometry_colorize $GEOMETRY_COLOR_PACKAGER_VERSION $GEOMETRY_SYMBOL_PACKAGER_VERSION) 
+
+# Package manager
+GEOMETRY_NODE_DEFAULT_PACKAGE_MANAGER="npm"
 
 geometry_prompt_node_setup() {}
 
 geometry_prompt_node_render() {
-    if (( $+commands[node] )); then
-        GEOMETRY_NODE_VERSION="$(node -v 2> /dev/null)"
-        GEOMETRY_NPM_VERSION="$(npm --version 2> /dev/null)" 
-        echo "$GEOMETRY_NODE_NPM_VERSION $GEOMETRY_NODE_VERSION ($GEOMETRY_NPM_VERSION)"
+    if [[ ! $+commands[node] ]]; then 
+        return
+    fi
+
+    if [[ ! -f "$PWD/package.json" ]]; then
+        return
     fi
+
+    if [[ $+commands[yarn] && -f "$PWD/yarn.lock" ]]; then
+        GEOMETRY_NODE_DEFAULT_PACKAGE_MANAGER="yarn"
+    fi 
+
+    GEOMETRY_PACKAGER_VERSION="$($GEOMETRY_NODE_DEFAULT_PACKAGE_MANAGER --version 2> /dev/null)" 
+    GEOMETRY_NODE_VERSION="$(node -v 2> /dev/null)"
+    echo "$GEOMETRY_NODE_PACKAGER_VERSION $GEOMETRY_NODE_VERSION ($GEOMETRY_PACKAGER_VERSION)"
```

Comment:
We can have a mark here to show which version are we displaying:

    ($GEOMETRY_NODE_DEFAULT_PACKAGE_MANAGER:$GEOMETRY_PACKAGER_VERSION)

It will look kind of: `v6.9.5 (npm: 3.10.10)`. Just a suggestion.